api/cmd/api/main.go

```diff
@@ -91,12 +89,39 @@ func main() {
 	logger.Info("server listening", slog.Int("port", Port))
 	// Generate openapi.yaml after the server starts
 	go func() {
+		// Wait for server start
 		time.Sleep(time.Millisecond * 250)
+
```

Comment:
Wait for server readiness check /ping before generating the new openapi.yaml and client SDK using orval.